hail/src/main/scala/is/hail/cxx/Compile.scala

```diff
@@ -1,15 +1,23 @@
 package is.hail.cxx
 
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+
+import is.hail.annotations.{Region, RegionValueBuilder}
 import is.hail.expr.ir
 import is.hail.expr.types.physical._
+import is.hail.io.CodecSpec
 import is.hail.nativecode.{NativeModule, NativeStatus, ObjectArray}
 import is.hail.utils.fatal
 
 import scala.reflect.classTag
 
 object Compile {
 
-  def makeNonmissingFunction(tub: TranslationUnitBuilder, body: ir.IR, args: (String, PType)*): (Function, Array[(String, NativeModule)]) = {
+  type Literals = (PTuple, CodecSpec => Array[Byte])
```

Comment:
I think having `Literals` be different types here and in `Emit` is confusing, and think the use in `Emit` makes the most sense. Is there a more specific name for what this type does?